fuse/dir.go

```diff
@@ -168,6 +168,13 @@ func (d *Dir) Lookup(
 	d.server.nodeDB[path] = &newNode
 	d.server.Unlock()
 
+	// Create the response
+	//
+	// XXX: how about the other fields of resp?
+	// resp.NodeID (must be unique)
+	// resp.Generation
+	resp.Attr = fuseAttrs
```

Comment:
I don't remember all the details but I believe there's no need to explicitly return these `fuseAttrs` as part of the `lookup()` file-operation. A `getAttr()` comes right after a `lookup()` to fetch these `fuseAttrs` that we are saving in the `NewFile` / `NewDir` instructions above.

Take a look at this code-path:

1) https://github.com/nestybox/fuse/blob/45cd9a3d884448418546d8eaa54ee7d772e576d6/fs/serve.go#L1057

2) https://github.com/nestybox/fuse/blob/45cd9a3d884448418546d8eaa54ee7d772e576d6/fs/serve.go#L1404

3) https://github.com/nestybox/fuse/blob/45cd9a3d884448418546d8eaa54ee7d772e576d6/fs/serve.go#L260

Reply:
In short, `file.Attrs()` instruction is indirectly utilized by several file-ops: `lookup()`, `mkdir()`, `create()`, `getAttr()`, etc.

Reply:
Technically, you can leave this new code here, but if you do, please add comments to explain what I said above (i.e., there's no real need for it taking into account bazil-fuse-lib's code structure).

Reply:
Ok got it, but I disagree a little bit :)

I think we should return the attributes in Lookup() because it's part of the [FUSE LookupResponse](https://pkg.go.dev/bazil.org/fuse@v0.0.0-20230120002735-62a210ff1fd5#LookupResponse). And I disagree we need to add a comment, because it's part of the interface to the FUSE library. The fact that FUSE happens to call GetAttr() right after it's an internal detail of the library which could change at any time.